clinica/pipelines/pet_surface/pet_surface_utils.py

```diff
@@ -1291,7 +1291,7 @@ def get_output_dir(is_longitudinal, caps_dir, subject_id, session_id):
         # Projection in native space
         (
             r"(.*(sub-.*)\/(ses-.*)\/pet\/surface)\/projection_native\/.*_hemi_([a-z]+).*",
-            r"\1/\2_\3_task-rest_acq-"
+            r"\1/\2_\3_trc-"
```

Comment:
More of a question than a comment but are we only handling "rest" tasks? It seems that in the specs this entity is indeed optional if the label is rest but would require a "corresponding [task events](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/05-task-events.html)" otherwise.

Reply:
You need the `task` entity on PET data if the latter were acquired _during_ a task-based acquisition. I don't think that's the case for ADNI and AIBL. We have got task-based fMRI, but PET was acquired under a different context.

Reply:
agree but a priori we are not limiting the use of `pet_surface` to `ADNI` and `AIBL`. But again not a critique of the changes per se but more of an open question for future reference.

Reply:
It's an excellent point, which highlights the inflexibility of the current search pattern. Previously, we would only consider PET data acquired in a task-based context, and only at resting state. Consequently, this pattern eliminates PET data from NIFD, OASIS and HABS.

The new one does not restrict us to task-based PET, which solves the issue for the new datasets. But indeed, with a proper abstraction, we could just query all PET imaging data for a given tracer.

Did I summarize your proposal correctly?

Reply:
Yes, clear. I'll add a comment somewhere in the BIDS compliant discussions to be attentive to this point.